HaulageApplication/HaulageTests/BillingTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using SQLite;
+using Haulage.DatabaseExecutionServices;
+using Haulage.BaseClasses.BillingHandler;
+using NuGet.Frameworks;
+
+namespace HaulageTests
+{
+    //public class BillingTests
+    //{
+        //private SQLiteConnection GetInMemoryConnection()
+        //{
+        //    var connection = new SQLiteConnection(":memory:");
+        //    return connection;
+        //}
+
+        //[Fact]
+        ////Unit test to get Bills from Database table
+        //public void GetBillFromDatabase()
+        //{
+        //    //Arrange
+        //    var connection = GetInMemoryConnection();
+        //    DatabaseSetup.InitializeDatabase();
```

Comment:
These commented-out code shouldn't be here.
It should either be deleted or uncommented

Reply:
deleted this and moved billing tests in here